src/main/java/com/stripe/param/TokenCreateParams.java

```diff
@@ -0,0 +1,2527 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class TokenCreateParams extends ApiRequestParams {
+  @SerializedName("account")
+  Account account;
+
+  @SerializedName("amount")
+  Long amount;
+
+  /** The bank account this token will represent. */
+  @SerializedName("bank_account")
+  Object bankAccount;
+
+  @SerializedName("card")
+  Object card;
+
+  @SerializedName("currency")
+  String currency;
+
+  /**
+   * The customer (owned by the application's account) for which to create a token. For use only
+   * with [Stripe Connect](https://stripe.com/docs/connect). Also, this can be used only with an
+   * [OAuth access token](https://stripe.com/docs/connect/standard-accounts) or [Stripe-Account
+   * header](https://stripe.com/docs/connect/authentication). For more details, see [Shared
+   * Customers](https://stripe.com/docs/connect/shared-customers).
+   */
+  @SerializedName("customer")
+  String customer;
+
+  @SerializedName("email")
+  String email;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  @SerializedName("external_id")
+  String externalId;
+
+  @SerializedName("iovation_blackbox")
+  String iovationBlackbox;
+
+  @SerializedName("ip")
+  String ip;
+
+  @SerializedName("payment_user_agent")
+  String paymentUserAgent;
+
+  @SerializedName("pii")
+  Pii pii;
+
+  @SerializedName("recipient")
+  String recipient;
+
+  @SerializedName("referrer")
+  String referrer;
```

Comment:
This is internal/sensitive field used by our SDKs like Stripe.js and should not leak in the library